plugins/modules/azure_rm_registrationdefinition.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Fred-Sun, (@Fred-Sun)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_registrationdefinition
+version_added: '1.3.0'
+short_description: Manage Azure RegistrationDefinition instance
+description:
+    - Create, update and delete instance of Azure RegistrationDefinition.
+options:
+    scope:
```

Comment:
does scope means the subscription in which the registration definition will be defined? If it is the case then it should be deleted, use the subscription provided by module parameter.

Reply:
I will delete it!